e2e/startupSmoke.spec.ts

```diff
@@ -38,6 +38,43 @@ class SacCalculatorPage {
     }
 }
 
+class GasPlannerPage {
+    private readonly bottomTimeInputSelector = '#duration';
+    private readonly addDiveButton = '#add-dive';
+    private readonly compareTabSelector = 'a.nav-link:has-text("Compare dives")';
```

Comment:
Selector not needed there is a route /diff so you can directly use the compre page address